dbt/adapters/duckdb/impl.py

```diff
@@ -2,6 +2,8 @@
 from dbt.adapters.sql import SQLAdapter
 from dbt.contracts.connection import AdapterResponse
 from dbt.exceptions import InternalException, RuntimeException
+from typing import Sequence
+from dbt.adapters.base.meta import available
```

Comment:
import ordering issue here-- I thought we had linting for this now tho?

Reply:
Yeah, we do. I had some problems with my Mac, so I used the other one. And did not install pre-commit. My bad

Reply:
Sorry, my bad, I thought that scaffolding template has import ordering in pre-commit config. Seems it is not true. So I reformated all files and added check into pre-commit. It is already merged into `master`